mints/03.json

```diff
@@ -0,0 +1 @@
+{"name":"contract/valory/fpmm_deterministic_factory:0.1.0","description":"The scaffold contract scaffolds a contract to be implemented by the developer.","code_uri":"ipfs://bafybeibveawsp3ir67otkyswyfozetesjwrhyt5gfix6fanjwfq4dd3vyy","image":"ipfs://bafybeiayofa7yqccdzg5zkyyccyybj5wzaclpkyaom45oogmmg2rbdwwly","attributes":[{"trait_type":"version","value":"0.1.0"}]}
```

Comment:
"The scaffold contract scaffolds a contract to be implemented by the developer." -> you need to update the metadata of this package in its yaml @jmoreira-valory